default_PanzerChasm.cfg

```diff
@@ -1,48 +1,61 @@
-"cd_volume" "0.5"
-"cl_color" "0"
-"cl_crosshair" "1"
-"cl_draw_fps" "1"
-"cl_fov" "90.000000"
-"cl_mouse_acceleration" "1"
-"cl_mouse_filter" "1"
-"cl_mouse_speed" "0.400000"
-"cl_name" "Player"
-"cl_old_style_perspective" "0"
-"cl_small_hud_mode" "0"
-"cl_weapon_reset" "0"
-"in_key_backward" "31"
-"in_key_forward" "35"
-"in_key_jump" "3"
-"in_key_left" "13"
-"in_key_look_down" "9"
-"in_key_look_up" "8"
-"in_key_right" "16"
-"in_key_turn_left" "10"
-"in_key_turn_right" "11"
-"in_reverse_mouse" "0"
-"r_animations_storage" "0"
-"r_brightness" "0.466667"
-"r_clear" "0"
-"r_debug_draw_depth_hierarchy" "0"
-"r_debug_draw_occlusion_buffer" "0"
-"r_dynamic_lighting" "0"
-"r_filter_hud_textures" "0"
-"r_filter_menu_textures" "0"
-"r_filter_textures" "0"
-"r_fullscreen" "0"
-"r_fullscreen_display" "0"
-"r_fullscreen_frequency" "60"
-"r_fullscreen_height" "640"
-"r_fullscreen_width" "480"
-"r_gl_vsync" "1"
-"r_msaa_level" "0"
-"r_shadows" "1"
-"r_software_gl_update_smooth" "0"
-"r_software_rendering" "1"
-"r_software_scale" "1"
-"r_software_use_gl_screen_update" "0"
-"r_window_height" "600"
-"r_window_width" "800"
-"s_nosound" "0"
-"s_volume" "0.500000"
-"sv_always_run" "1"
+"cd_volume" "0.500000"

```

Comment:
Let's avoid making unnecessary changes in this file. It would be better to add default values for new settings manually, rather just replacing the config file with newly-generated.